hw2/04-background.js

```diff
@@ -1 +1,55 @@
-// Add your code here
+const startButton = document.getElementById('startBtn');
+
+// A variable to store the interval ID
+let intervalId = null;
+
+// Function to toggle button appearance and functionality
+const toggleButton = () => {
+  if (startButton.textContent === 'Start') {
+    startButton.className = 'btn btn-danger';
+    startButton.textContent = 'Stop';
+    return true;
+  } else {
+    startButton.className = 'btn btn-primary';
+    startButton.textContent = 'Start';
+    return false;
+  }
+};
+
+// Function to get a random integer
+const getRandomInt = (max) => Math.floor(Math.random() * Math.floor(max));
+
+// Function to change the background color of the body element
+const changeBackgroundColor = () => {
```

Comment:
instead of adding the forth argument of rgba as a const, adding it as the function argument could make it more usable.